Added contacts feature, documented in core/contacts/README.md. (#1706)
talonhub/community: core/contacts/README.md

Author: If preferred, I could add this section directly to the top-level readme, or link to it from there.

# Contacts

This directory provides a versatile `<user.prose_contact>` capture that can be
used to insert names and email addresses using a suffix. The contact list may be

> provided through `contacts.json` in this directory, `contacts.csv` in the
> settings directory, or both.

Reviewer: ```suggestion
provided in the private directory via the `contacts.json` file, the `contacts.csv` file,  or both.
```
I believe as written these both need to live in the private directory?
